src/app/components/cds/media-cards/ItemReportStatus.js

```diff
@@ -35,10 +41,18 @@ const ItemReportStatus = ({
     );
   };
 
+  const handleGoToReport = () => {
+    if (projectMediaDbid) {
+      console.log('projectMediaDbid:', projectMediaDbid); // eslint-disable-line no-console
```

Comment:
@amoedoamorim please remove this before merging

Reply:
oopsie!